common/mercenaries/wc_mercenaries.txt

```diff
@@ -2,180 +2,169 @@
+	levy_size = 2
 	heavy_infantry = 600
 	archers = 300
+	siege_machines = 100
 	# galleys = 40
 }
 dogs_of_war_composition = {
-	levy_size = 1
-	siege_machines = 50
-	heavy_infantry = 500
-	light_cavalry = 200
+	levy_size = 3
+	heavy_infantry = 550
+	light_cavalry = 175
 	archers = 250
+	siege_machines = 50
 }
 stormriggers_composition = {
-	levy_size = 1
-	heavy_infantry = 500
-	archers = 250
+	levy_size = 3
+	archers = 825
+	elite_infantry = 175
 }
 highborne_company_composition = {
 	levy_size = 1
-	heavy_infantry = 500
-	light_infantry = 200
+	light_infantry = 650
+	elite_infantry = 350
 }
 centaur_composition = {
 	levy_size = 1
-	light_cavalry = 500
-	large_cavalry = 75
-	archers = 300
+	light_cavalry = 400
+	horse_archers = 600
 }
 troll_composition = {
-	levy_size = 1
-	heavy_infantry = 700
-	archers = 300
-	light_infantry = 50
+	levy_size = 3
+	light_infantry = 100
+	heavy_infantry = 475
+	pikemen = 100
+	light_cavalry = 100
+	archers = 225
 }
 korkron_composition = {
-	levy_size = 2
-	heavy_infantry = 650
-	elite_infantry = 75
-	light_cavalry = 250
+	levy_size = 4
+	heavy_infantry = 250
+	light_cavalry = 150
+	archers = 450
+	elite_infantry = 150
 }
 taunka_composition = {
-	levy_size = 1
-	heavy_infantry = 700
-	archers = 100
-	light_infantry = 100
+	levy_size = 2
+	light_infantry = 200
+	pikemen = 500
+	archers = 300
 }
 vrykul_composition = {
-	levy_size = 3
-	pikemen = 200
-	heavy_infantry = 250
-	archers = 250
-	light_cavalry = 100
+	levy_size = 2
+	light_infantry = 300
+	pikemen = 450
+	archers = 225
+	dragons = 25
 	# galleys = 80
 }
 frost_vrykul_composition = {
-	levy_size = 1.5
-	heavy_infantry = 500
-	pikemen = 200
-	archers = 300
+	levy_size = 2.5
+	pikemen = 350
+	archers = 550
+	elite_infantry = 100
 }
 wolvar_composition = {
 	levy_size = 1.5
-	heavy_infantry = 450
-	archers = 100
+	heavy_infantry = 300
+	archers = 400
+	thralls = 300
 }
 gorloc_composition = {
-	levy_size = 2
-	heavy_infantry = 500
-	light_infantry = 50
+	levy_size = 1.5
+	light_infantry = 375
+	heavy_infantry = 625
 }
 tuskarr_composition = {
-	levy_size = 1
-	heavy_infantry = 350
-	light_cavalry = 150
-	archers = 100
-}
-northern_composition = {
-	levy_size = 2
-	heavy_infantry = 500
-	light_cavalry = 100
-	archers = 400
+	levy_size = 1.5
+	light_infantry = 100
+	heavy_infantry = 550
+	archers = 300
+	large_cavalry = 50
 }
 goblin_composition = {
-	levy_size = 1
-	archers = 100
+	levy_size = 2.3
 	heavy_infantry = 200
-	knights = 150
-	war_machines = 100
+	archers = 650
+	war_machines = 150
 }
 snobold_composition = {
-	levy_size = 1
-	heavy_infantry = 700
-	archers = 200
-	light_infantry = 100
+	levy_size = 2
+	light_infantry = 225
+	heavy_infantry = 175
+	thralls = 600
 }
 naval_merc_composition = {
 	levy_size = 9
 	galleys = 10
 }
 cenarion_hold_composition = {
-	levy_size = 4
-	heavy_infantry = 350
-	light_infantry = 100
-	archers = 150
-	animals = 250
+	levy_size = 2
+	light_infantry = 400
+	animals = 350
+	elite_infantry = 250
```

Comment:
you can't have 2 special units in one composition

Reply:
Fixed!